app/renderer/js/electron-bridge.js

```diff
@@ -32,6 +32,10 @@ electron_bridge.on('realm_icon_url', iconURL => {
 	ipcRenderer.send('realm-icon-changed', serverURL, iconURL);
 });
 
```

Comment:
Maybe add a few comments here about what's going on so that others can also understand?